console/config/schema.json

```diff
@@ -1273,6 +1273,10 @@
           "type": "map", 
           "description": "A map where each key is a user group name and the value is a map of the corresponding settings for that group."
         }, 
+        "hostname": {
```

Comment:
Could this use the deprecationName attribute from my earlier PR - https://github.com/apache/qpid-dispatch/pull/261/files#diff-981a10a2c9cab3708f003b402e7f8be4R1060

Reply:
Can't be used until you check it in...

Reply:
yes, agreed